pkg/controller/cloud/node_lifecycle_controller_test.go

```diff
@@ -258,10 +286,20 @@ func Test_NodesDeleted(t *testing.T) {
 			eventBroadcaster.StartLogging(klog.Infof)
 			cloudNodeLifecycleController.MonitorNodes()
 
-			if !reflect.DeepEqual(testcase.fnh.DeletedNodes, testcase.deleteNodes) {
-				t.Logf("actual nodes: %v", testcase.fnh.DeletedNodes)
-				t.Logf("expected nodes: %v", testcase.deleteNodes)
-				t.Error("unexpected deleted nodes")
+			updatedNode, err := clientset.CoreV1().Nodes().Get(context.TODO(), testcase.existingNode.Name, metav1.GetOptions{})
+			if err != nil {
```

Comment:
Would it be easier to have a `expectDeleted` field in the test table and we can compare that to the value of `IsNotFound(err)`?